cmd/root.go

```diff
@@ -98,5 +98,5 @@ func Execute() {
 func init() {
 	RootCmd.PersistentFlags().StringVarP(&configFile, "config", "c", "checkup.json", "JSON config file")
 	RootCmd.Flags().BoolVar(&storeResults, "store", false, "Store results")
-	RootCmd.Flags().BoolVar(&printLogs, "v", false, "Enable logging to standard output")
+	RootCmd.Flags().BoolVarP(&quiet, "quiet", "q", false, "Disable logging to standard output")
```

Comment:
This flag is documented as:

> Disable logging to standard output

I don't think that accurately describes what it does. I think it should be:

> Disable logging to standard error

Because the default output of the standard logger in `log` package is `os.Stderr`, not `os.Stdout`:

> Package log implements a simple logging package. It defines a type, Logger, with methods for formatting output. It also has a predefined 'standard' Logger accessible through helper functions Print[f|ln], Fatal[f|ln], and Panic[f|ln], which are easier to use than creating a Logger manually. **That logger writes to standard error** and prints the date and time of each logged message.

Reply:
Good point.

Maybe we shouldn't confuse the user with stdout or stderr anyway, maybe it should just say "Do not display log messages"?

Reply:
There is an inconsistent use of log/fmt, BTW.

For example, those fmt.Print should probably be log.* calls:
https://github.com/sourcegraph/checkup/blob/33d13c4b28611d74d090162ff891a4058910cc80/cmd/message.go#L76
https://github.com/sourcegraph/checkup/blob/33d13c4b28611d74d090162ff891a4058910cc80/cmd/provision.go#L61

So the flag won't make checkup totally "mute" anyway.

Reply:
The problem is we need to distinguish between logging and "standard output" -- the CLI is interactive (well, sorta -- you give the input via env vars and/or flags) and so the fmt.Print lines are supposed to show those things so the user knows what's going on. The logging output is something else, so maybe "quiet" is too ambiguous. What about instead, the main package disables logging by default unless logging is turned on.

Reply:
I still think that the user doesn't *need* to see "One sec" but, OTOH, the user would need to see a fatal configuration parse error.  So IMHO something's wrong with the current logging system.

Anyways, so you suggest we keep current behavior?

Reply:
Good point... there are also occasions where the output is credentials that are generated by a provisioning command and the user only has 1 shot at getting those, so it's critical that it can't be turned off.

Maybe the configuration errors need to be fmt.Print'ed instead of logged.

Reply:
Errors could also be sent to stderr with *fmt.Fprint*, so that they would not be in the data stream when checkup is called from a script.

However, the checkup "library" should not send anything directly to stdout/stderr by itself.